pkg/validate/dynamic/dynamic_test.go

```diff
@@ -1727,3 +1728,39 @@ func TestCheckBYONsg(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateSubnetSize(t *testing.T) {
+	subnetId := "id"
+	subnetPath := "path"
+	for _, tt := range []struct {
+		name    string
+		address string
+		subnet  Subnet
+		wantErr string
+	}{
+		{
+			name:    "subnet size is too small",
+			address: "10.0.0.0/32",
+			subnet:  Subnet{ID: subnetId, Path: subnetPath},
+			wantErr: fmt.Sprintf("400: InvalidLinkedVNet: %s: The provided subnet '%s' is invalid: must be /27 or larger.", subnetPath, subnetId),
+		},
+		{
+			name:    "subnet size is gucci gang",
+			address: "10.0.0.0/27",
+			subnet:  Subnet{ID: subnetId, Path: subnetPath},
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			err := validateSubnetSize(tt.subnet, tt.address)
+			if err == nil {
+				if tt.wantErr != "" {
+					t.Error(err)
+				}
+			} else {
+				if err.Error() != tt.wantErr {
+					t.Error(err)
+				}
+			}
```

Comment:
This can be replaced by
```go
utilerror.AssertErrorMessage(t, err, tt.wantErr)
```

Reply:
done - good catch i didn't know that was a thing.